README.rst

```diff
@@ -52,6 +52,8 @@ howdoi will answer all sorts of queries:
 
     $ howdoi create tar archive
     > tar -cf backup.tar --exclude "www/subf3" www
+    
+.. image:: howdoi.gif
```

Comment:
Not actually a request, but I feel like this is not directly related to the json issue. (it's fine though)

Reply:
Whoops -- yeah you're definitely right about that, my bad 😅